admin/environment-management/workspace-providers.md

```diff
@@ -4,17 +4,35 @@ description: Learn how workspace providers can minimize latency for users.
 state: beta
 ---
 
-Workspace Providers are a forthcoming feature that enables a single Coder
-deployment to provision and manage workspaces across multiple Kubernetes
-clusters and namespaces, including ones located in other geographies, regions,
-or clouds.
+Workspace Providers are logical groups of resources developers can deploy
+workspaces onto. Workspace Providers enable a single Coder deployment to
+provision and manage workspaces across multiple Kubernetes clusters and
+namespaces, including ones located in other geographies, regions, or clouds.
 
 Distributed teams can use this feature to allow users to manage workspaces in
 the cluster that's nearest, reducing network latency and improving developers'
 experience. Additionally, you can use workspace providers to support data
 sovereignty requirements or increase the isolation between workspaces running in
 the same region or cluster.
 
+### Built-In Workspace Provider
+
+By default, all Coder deployments will have a `built-in` Workspace Provider that
+identifies the kubernetes cluster the Coder deployment is in. This enables
```

Comment:
```suggestion
specifies the Kubernetes cluster that contains the Coder deployment. This enables
```

Just a general thought, but perhaps we could think of "Coder Deployment" as a logical thing encompassing both our control plane (CEManager, etc) as well as our data plane (all running Workspaces/Environments). So what we're really saying is that the built-in Workspace Provider identifies the Kubernetes cluster that is hosting the Coder control plane itself, whereas the workspaces and workspace providers are part of our "data plane"?

The distinction is used in telecom to differentiate between elements that are in the critical path (e.g. switches that handle every packet, aka data plane, where performance is absolutely critical) versus elements that are outside the critical path (e.g. management of the data plane layer aka control plane, which is outside the critical path and can therefore be slower/more compute intensive).

[Cloudflare explains it with this analogy](https://www.cloudflare.com/learning/network-layer/what-is-the-control-plane/):

>Think of the control plane as being like the stoplights that operate at the intersections of a city. Meanwhile, the data plane (or the forwarding plane) is more like the cars that drive on the roads, stop at the intersections, and obey the stoplights.